hyprpm/src/core/PluginManager.cpp

```diff
@@ -505,48 +509,46 @@ bool CPluginManager::updateHeaders(bool force) {
         missing             = missing.substr(0, missing.find("-- Configuring incomplete"));
         missing             = missing.substr(0, missing.find_last_of('\n'));
 
-        std::cerr << "\n"
-                  << Colors::RED << "✖" << Colors::RESET << " Could not configure the hyprland source, cmake complained:\n"
-                  << missing << "\n\nThis likely means that you are missing the above dependencies or they are out of date.\n";
+        std::println(stderr, "\n{}",
+                     failureString("Could not configure the hyprland source, cmake complained:\n{}\n\n"
+                                   "This likely means that you are missing the above dependencies or they are out of date.",
+                                   missing));
         return false;
     }
 
-    progress.printMessageAbove(std::string{Colors::GREEN} + "✔" + Colors::RESET + " configured Hyprland");
+    progress.printMessageAbove(successString("configured Hyprland"));
     progress.m_iSteps           = 4;
     progress.m_szCurrentMessage = "Installing sources";
     progress.print();
 
-    std::string cmd =
+    const std::string& cmd =
         std::format("sed -i -e \"s#PREFIX = /usr/local#PREFIX = {}#\" {}/Makefile && cd {} && make installheaders", DataState::getHeadersPath(), WORKINGDIR, WORKINGDIR);
     if (m_bVerbose)
-        progress.printMessageAbove(std::string{Colors::BLUE} + "[v] " + Colors::RESET + "installation will run: " + cmd);
+        progress.printMessageAbove(verboseString("installation will run: " + cmd));
```

Comment:
considering you made those take a format string wouldn't it be nice to also get rid of those and just `{}` them? :)

Reply:
With pleasure

Reply:
So make them 
```c++
progress.printMessageAbove(verboseString("installation will run: {}", cmd));
```
Instead right?

Reply:
ya